src/Uno.UI.RemoteControl.Server.Processors/HotReload/ServerHotReloadProcessor.MetadataUpdate.cs

```diff
@@ -61,11 +61,21 @@ private void InitializeInner(ConfigureServer configureServer) => _initializeTask
 				{
 					await Notify(HotReloadEvent.Initializing);
 
+					var properties = configureServer
+						.MSBuildProperties
+						.ToDictionary();
+
+					properties["UnoIsHotReloadHost"] = "True";
+					if (properties.Remove("RuntimeIdentifier", out var runtimeIdentifier))
```

Comment:
```suggestion
                    // Set the RuntimeIdentifier as a temporary property so that we do not force the
                    // property as a read-only global property that would be transitively applied to
                    // projects that are not supporting the head's RuntimeIdentifier. (e.g. an android app
                    // which references a netstd2.0 library project)
					if (properties.Remove("RuntimeIdentifier", out var runtimeIdentifier))
```